### PR TITLE
Window title / `world.name`

### PR DESCRIPTION
**Pull request by ZeWaka**
title

it goes `world.name` -> the filename of the .dme -> i don't think its possible to go past that but i put a fallback

### CODE REVIEW

**Comment by ZeWaka** [comment]
Ideally the worldName finding stuff would live somewhere else and be stored in some global state, but we're trying to get rid of global states so I'm not sure what I would do to remedy.

File: OpenDreamRuntime/DreamRuntime.cs [hunk 1]
```diff
@@ -222,16 +224,27 @@ public class DreamRuntime
             connection.SendPacket(new PacketInterfaceData(_clientInterface));
             connection.SendPacket(new PacketFullGameState(StateManager.FullState));
 
+            // Set to <world> part of <world>.json unless we define world.name - final fallback to "OpenDream World"
+            String worldName;
+            DreamValue worldInstanceName = WorldInstance.GetVariable("name");
+            if (worldInstanceName.Value != null ) {
```
Comment (wixoaGit) [inline OpenDreamRuntime/DreamRuntime.cs:230]:
```suggestion
            if (worldInstanceName != DreamValue.Null) {
```

**Review by wixoaGit — COMMENTED** [review]
This doesn't handle `/world.name` changing while the client is already connected

**Comment by ike709** [comment]
...How's this going?

**Comment by ZeWaka** [comment]
Haven't gotten to it - feel free to take it over, IRL has been busy and I have a deadline for goon

I also don't know how I would handle this for the stated error above, however I forget beyond behavior, but personally I don't think that the window title would change upon update unless you reconnect in which case my PR would work

**Comment by ZeWaka** [comment]
Probably irrelevant with RobustToolbox

**Pull request closed without merging**